src/librustc/diagnostics.rs

```diff
@@ -1962,6 +1962,19 @@ each method; it is not possible to annotate the entire impl with an `#[inline]`
 attribute.
 "##,
 
+E0522: r##"
+A new lang item was created in user code, which is forbidden. Erroneous code example:
```

Comment:
Perhaps: "The lang attribute is intended for marking special items that are built-in to Rust itself. This includes special traits (like `Copy` and `Sized`) that affect how the compiler behaves, as well as special functions that may be automatically invoked (such as the handler for out-of-bounds accesses when indexing a slice)."